.rubocop.yml

```diff
@@ -130,6 +130,15 @@ Style/FrozenStringLiteralComment:
     - 'actionpack/test/**/*.ruby'
     - 'activestorage/db/migrate/**/*.rb'
 
+Style/RedundantFreeze:
+  Enabled: true
+  Exclude:
+    - 'actionpack/test/controller/test_case_test.rb'
+    - 'activemodel/test/cases/type/string_test.rb'
+    - 'activesupport/test/core_ext/string_ext_test.rb'
+    - 'activesupport/lib/active_support/core_ext/string/strip.rb'
```

Comment:
This is not redundant freeze.
https://github.com/rails/rails/blob/d03cc170c1bdad15245be7f2681a265fd2bd0d5d/activesupport/lib/active_support/core_ext/string/strip.rb#L24

Reply:
This `"".freeze` is obviously redundant, there is no reason to exclude it.
https://github.com/rails/rails/blob/d03cc170c1bdad15245be7f2681a265fd2bd0d5d/activesupport/lib/active_support/core_ext/string/strip.rb#L23